packages/affiliate/src/Affiliate.php

```diff
@@ -1,25 +1,33 @@
 <?php
+namespace Automattic\Jetpack\Partners;
 
-if ( ! defined( 'ABSPATH' ) ) {
-	// Exit if accessed directly
-	exit;
+use Jetpack;
+
+if ( ! defined( 'ABSPATH' ) || ! is_admin() ) {
+	exit; // Exit if accessed directly or not in admin.
 }
 
 /**
  * This class introduces routines to get an affiliate code, that might be obtained from:
- * - an `jetpack_affiliate_code` option in the WP database
+ * - a `jetpack_affiliate_code` option in the WP database
  * - an affiliate code returned by a filter bound to the `jetpack_affiliate_code` filter hook
  *
  * @since 6.9.0
```

Comment:
Noting that once we move this code to a package, such `@since` references become ambigious, because it's not certain whether this is the Jetpack version or the package version.

Reply:
All removed.